src/ch/blinkenlights/android/vanilla/LibraryActivity.java

```diff
@@ -332,19 +332,15 @@ public boolean onKeyUp(int keyCode, KeyEvent event)
 	}
 
 	@Override
-	public boolean onKeyDown(int keyCode, KeyEvent event)
-	{
+	public boolean onKeyDown(int keyCode, KeyEvent event) {
```

Comment:
Here you change brackets to be on the same line as definition but further in your PR (e.g. in `buildPlaylistQuery`) you keep them on next line. Either fix this one or restyle everything.